packages/venia-concept/src/components/AppShell/renderRoutes.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { Page } from '@magento/peregrine';
+import ErrorView from 'src/components/ErrorView';
+import CreateAccountPage from 'src/components/CreateAccountPage';
+
+const renderRoutingError = props => <ErrorView {...props} />;
+
+const renderRoutes = () => (
+    <Switch>
+        <Route exact path="/create-account" component={CreateAccountPage} />
+        <Route render={() => <Page>{renderRoutingError}</Page>} />
+    </Switch>
```

Comment:
Are we certain that it's safe to intercept `/create-account` and define it as a client-side route? Afaik, routing is owned by the Magento instance.

Also, we already have a Create Account workflow in the left drawer. Why do we need a new page with the same workflow?

Reply:
I discussed the question about create separate route for create account page with James, we decided to implement this for now.
Out current Create Account workflow is bad from SEO perspective, it is important to have direct urls for Create Account and Sign In pages.
Initially the problem was to navigate user from order confirmation step of checkout in the right drawer to create account page in the left drawer. Current implementation does not allow to do that, because we manage drawer state locally.
@zetlen proposed to implement a routed drawer, but this solution requires a lot of effort to implement it.
Just to summarize, in this pull request I added code to implement business requirement with navigating user to Create Account page from Order Confirmation and this separate Create Account page is better from SEO perspective.

Reply:
Okay.

This does satisfy the business requirement of navigating the user after confirmation. I would prefer to open the left drawer to Create Account, but this approach of rendering a new page is acceptable for now. We can treat this approach as tech debt and have someone else revisit it later as part of a broader look at routing and transitions.